src/_scss/pages/covid19/index.scss

```diff
@@ -0,0 +1,339 @@
+              .results-table-message-container {
+                  position: relative;
+                  background-color: rgb(246, 246, 246);
+                  .results-table-loading {
+                      margin-bottom: rem(100);
+                  }
+                  .no-results-container {
+                      text-align: center;
+                      margin-top: rem(100);
+                      margin-bottom: rem(100);
+                      .no-results-title {
+                          h4 {
+                          margin-bottom: 0;
+                          vertical-align: bottom;
+                          }
+                      }
+                      .no-results-message {
+                          p {
+                          margin-top: 0;
+                          vertical-align: top;
+                          }
+                      }
+                      }
+                      .loading-animation {
+                          @import "../search/results/screens/loading";
+                      }
+                      .results-table-error {
+                      .icon {
+                          width: rem(36);
+                          height: rem(36);
+                          svg {
+                          width: rem(36);
+                          height: rem(36);
+                          }
+                      }
+                      .title {
+                          font-size: rem(20);
+                      line-height: rem(45);
+                      }
+                      .description {
+                          font-size: rem(16);
+                          line-height: rem(10);
+                      }
+                  }
+              }
+              .usda-table {
+                  td {
+                      &:first-child {
+                          @include span-columns(6 of 12, table);
+                          text-align: left;
+                      }
+                      @include span-columns(2 of 12, table);
+                      text-align: right;
+                      word-break: break-word;
+                  }
+
+                  .table-header {
+                      .table-header__label {
+                          @include flex(0 1 auto);
+                          text-align: right;
+                      }
+  
+                      .table-header__sort {
+                          margin-top: auto;
+                          margin-bottom: auto;
+                      }
+                  }
+              }
+          }
+          .body__section {
+              @import '../agency/v2/tabs';
+              @include display(flex);
+              @include flex-wrap(wrap);
+              margin-bottom: $global-mrg;
+              .body__header {
+                  width: 100%;
+                  display: flex;
+                  flex-wrap: wrap;
+                  align-items: center;
+                  justify-content: space-between;
```

Comment:
`@include display(flex)`
`@include flex-wrap(wrap)`
etc...

Reply:
I thought we were moving away from @include because it was part of bourbon or something?

Reply:
I was not aware. Please disregard my comments then!

Reply:
same as my other comment below